README.md

```diff
@@ -13,7 +13,9 @@ Axiom lets you make the most of your event data without compromises: all your da
 ## Prerequisites
 
 - [Sign up for a free Axiom account](https://app.axiom.co/register). All you need is an email address.
-- [Create an API token in Axiom with permissions to query and ingest data](https://axiom.co/docs/reference/settings#access-overview).
+- [Create an advanced API token in Axiom](https://axiom.co/docs/reference/tokens#create-advanced-api-token) with the permissions to perform the actions you want to use Terraform for. For example, to use Terraform to create and update datasets, create the advanced API token with these permissions.
+- [Create a Terraform account](https://app.terraform.io/signup/account).
```

Comment:
Why is this included?

Reply:
@Rambatino Thanks a lot for your review! I accept the suggestions. I would like to keep the prerequisite about a Terraform account. Can this procedure function without it?

Reply:
Yeah I've never signed up and I don't think we should suggest that people need to sign up to a service unless they have to (unless something has changed, but I doubt it)

Reply:
Didn't you have to run `terraform login` at some point? It usually stores the token in `~/.terraform.d/credentials.tfrc.json` and then you don't need to worry about it again.